src/contexts/Note/NoteReducer.jsx

```diff
@@ -0,0 +1,67 @@
+export const noteReducer = (state, { type, payload }) => {
+  const intialState = {
+    notes: [],
+    archivedNotes: [],
+    trashedNotes: [],
+    pinnedNotes: [],
+  };
+  switch (type) {
+    case "GET_NOTE":
+      return { ...state, notes: payload };
+    case "GET_ARCHIVED_NOTES":
+      return { ...state, archivedNotes: payload };
+    case "ADD_NOTE":
+      return { ...state, notes: payload };
```

Comment:
Single case be made for all three as all are doing same action

Reply:
Yes, thank you. Have corrected it.